test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -156,6 +158,7 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.CyclicBarrier;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
```

Comment:
I see new imports, but they don't seem used?

Reply:
ha :) that's the previous hack to trigger the issue. need to remove these. strange that checkstyle does not complain.